### DISCUSSION TITLE
Bug Report: FlxBar: max cannot be less than or equal to min

### DISCUSSION THREAD

**Discussion by willboyz92nd** [KadeArchive/Kade-Engine#467] [Q&A] [answered]
So someone asked me to chart a song for them when they sent me the song and i loaded it up i got this error or the notes would bug out

**Comment by isakube** [comment]
https://github.com/KadeDev/Kade-Engine/discussions/423

**Comment by willboyz92nd** [comment]
for everyone having this issue i found the fix when you use a song make sure you use a converter tool to convert mp3 to ogg just renaming the file to .ogg will give you the error

**Reply by myst-rblx** [reply]
it didnt work

**Comment by willboyz92nd** [comment, marked as answer]
for everyone having this issue i found the fix when you use a song make sure you use a converter tool to convert mp3 to ogg just renaming the file to .ogg will give you the error make sure to use a converter tool like https://www.zamzar.com/convert/mp4-to-ogg/ if you just rename the file you will still crash
